adobe_vipm/flows/fulfillment/change.py

```diff
@@ -346,6 +347,7 @@ def fulfill_change_order(client, order):
         GetPreviewOrder(),
         UpdatePrices(),
         SubmitNewOrder(),
+        NullifyFlexDiscountParam(),
```

Comment:
You should do it after the order is completed. Otherwise after the order completion the agreement parameter value is going to be override with the order value.

So I would just place it right after the `CompleteOrder` step

Reply:
DOne